ehrenamt-justiz-backend/README.md

```
@@ -177,4 +177,24 @@ API-docs can be called from the backend if the profile 'local' is used:
 http://localhost:39146/v3/api-docs
 ```
 
-![ER Model](../docs/images/ApiDocs_Backend.PNG)
+![API-docs](../docs/images/ApiDocs_Backend.PNG)
+
+# Logging
```

markdownlint-cli2 (0.17.2)

Multiple top-level headings in the same document

(MD025, single-title, single-h1)